rpc/dialer.go

```diff
@@ -68,6 +73,15 @@ type ClientConnAuthenticator interface {
 	Authenticate(ctx context.Context) (string, error)
 }
 
+// A GrpcOverHttpClientConn is grpc connection that is not backed by a `webrtc.PeerConnection`.
+type GrpcOverHttpClientConn struct {
```

Comment:
This is public because there's a place in the RDK that does a direct grpc dial that I need to instrument. I figured it'd be appropriate to reuse this. Even if the type is buried pretty deep.